Sources/XcodeGenKit/ProjectGenerator.swift

```diff
@@ -33,7 +33,16 @@ public class ProjectGenerator {
         return spec.configs.first { $0.type == .release }!
     }
 
-    func validate() throws {
+    func getPath(_ path: String) -> Path {
```

Comment:
There's a similar method in `PBXProjGenerator`. Have you consider extracting it somewhere to use it from both places?